includes/rules.php

```diff
@@ -74,6 +74,14 @@
 			'<code>&lt;h2&gt;</code>'
 		),
 	),
+	array(
+		'title'     => esc_html__( 'Empty Paragraph Tag', 'accessibility-checker' ),
+		'info_url'  => 'https://a11ychecker.com/',
+		'slug'      => 'paragraph_should_not_be_empty',
```

Comment:
Can we change this to `empty_paragraph_tag ` to be consistent with other rule slugs?